wallet/lib/pages/home/currency_screen/currency_screen.dart

```diff
@@ -1,366 +1,365 @@
-    }
-  }
-
-  Future<void> handleStripeAccountLink() async {
-    final loading = Loading()..showLoading();
-
-    final account_response = await GetIt.I.get<StripeHandler>().handleStripeAccountLink();
-    loading.dismiss();
-    account_response.fold((fail) => {fail.message.show()}, (accountlink) {
-      showDialog(
-          useSafeArea: false,
-          context: context,
-          builder: (BuildContext context) {
-            return StripeScreen(
-                url: accountlink,
-                onBack: () {
-                  navigatorKey.currentState!.pop();
-                });
-          });
-    });
-  }
-}
-
-class BalanceIBCCoins extends StatefulWidget {
-  final Balance balance;
-  final IBCCoins ibcCoins;
-
-  const BalanceIBCCoins({Key? key, required this.balance, required this.ibcCoins}) : super(key: key);
-
-  @override
-  _BalanceIBCCoinsState createState() => _BalanceIBCCoinsState();
-}
-
-class _BalanceIBCCoinsState extends State<BalanceIBCCoins> {
-  @override
-  void initState() {
-    super.initState();
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    return Card(
-        shape: RoundedRectangleBorder(
-          borderRadius: BorderRadius.circular(20.0),
-        ),
-        margin: const EdgeInsets.symmetric(horizontal: 20, vertical: 10),
-        borderOnForeground: false,
-        elevation: 20,
-        child: Container(
-          padding: const EdgeInsets.symmetric(horizontal: 20, vertical: 17),
-          width: 1.sw,
-          height: 0.35.sw,
-          decoration: BoxDecoration(
-              borderRadius: BorderRadius.circular(20.0),
-              color: denomColors[widget.balance.denom] ?? Colors.blueGrey,
-              image: DecorationImage(
-                  image: const AssetImage('assets/images/masks/card_luma.png'),
-                  fit: BoxFit.fill,
-                  colorFilter: ColorFilter.mode(denomColors[widget.balance.denom] ?? Colors.blueGrey, BlendMode.overlay))),
-          child: Column(
-            mainAxisAlignment: MainAxisAlignment.spaceBetween,
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: <Widget>[
-              Row(children: [
-                widget.ibcCoins.getAssets(),
-                SizedBox(
-                  width: 10.w,
-                ),
-                Text(
-                  widget.ibcCoins.getName(),
-                  style: Theme.of(context).textTheme.subtitle1!.copyWith(color: Colors.white, fontSize: 18),
-                ),
-                const Spacer(),
-              ]),
-              Align(
-                alignment: Alignment.centerRight,
-                child: Text(
-                  "\$${"${widget.balance.amount.toHumanReadable()}".trimZero()}",
-                  style: Theme.of(context).textTheme.subtitle1!.copyWith(color: Colors.white, fontSize: 24, fontFamily: 'Inter'),
-                ),
-              ),
-            ],
-          ),
-        ));
-  }
-}
+// import 'dart:async';
```

Comment:
Instead of commenting we should delete it.

Reply:
This change was committed by your end. So thats why I didn't remove that.
Deleted now.